### PR TITLE
Updated Pre-Commit.md

### LINKED ISSUE #592: Update steps for installing and using pre commit
Update the PRE-COMMIT.md file. Currently, specific instructions are not present.

Add the instructions for installing pre commit and running it in a repo

### PR DESCRIPTION
**Pull request by Salonichoudhary07**
Fixes #592 

## Summary
Commands for Pre-commit installation and usage are added in the doc.

## Test Plan

- [ ] <strike>Test Responsiveness
   - [ ] Laptop (1200px)
   - [ ] Tablet (760px)
   - [ ] Phone (320px)
   </strike>
- [ ] <strike>Cross-Browser Testing
   - [ ] Chrome
   - [ ] Firefox
   - [ ] Safari (use BrowserStack)
   </strike>
- [x] Hygiene and Housekeeping
   - [x] Self-review
   - [x] Comments have been added appropriately
   - [ ] <strike>Check for bundle size [here](https://bundlephobia.com/) if adding a package</strike>
   - [ ] <strike>Added relevant details like Labels/Projects/Milestones etc.</strike>
   - [ ] <strike>If adding or removing any environment variable:
       - [ ] update `docs/ENV.md` 
       - [ ] update Github Workflow files
       - [ ] update DockerFile
       - [ ] update the secrets for staging and production
       </strike>
- [ ] Testing
   - [ ] ~Wrote tests~
   - [x] Tested locally
   - [ ] Tested on staging
   - [ ] ~Tested on an actual physical phone~
   - [ ] Tested on production
- [ ] <strike> Lighthouse Tests
   - [ ] Images have `alt` attributes
   - [ ] Any `` tags have `width` and `height` specified
   - [ ] Any `target="_blank"` links have `rel="noopener"`
   - [ ] Only SVGs are used as images. If PNGs are used, their size has been optimised.
   - [ ] Any SVGs without text have their `aria-label` attributes set
   </strike>

### CODE REVIEW

File: docs/PRE-COMMIT.md [hunk 1]
```diff
@@ -10,3 +10,13 @@ The pre-commit hook in this repository uses various plugins to run different kin
...
+
+#Installation
+Install pre-commit package manager.
+    pip install pre-commit
+
+You can install it in your codespace using this: https://github.com/avantifellows/plio-frontend/blob/master/docs/PRE-COMMIT.md.
+
+#Usage
+Run command:
+    pre-commit .
```
Comment (dalmia) [inline docs/PRE-COMMIT.md:22]: please look at `docs/INSTALLATION.md` for how to correctly set headings (this has no space between `#` and the text) - did you actually see how it looks in the markdown file?

File: docs/PRE-COMMIT.md [hunk 2]
```diff
@@ -10,3 +10,13 @@ The pre-commit hook in this repository uses various plugins to run different kin
 1. [Pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks): Checks and fixes basic linting issues. All available hooks for pre-commit can be found [here](https://github.com/pre-commit/pre-commit-hooks#hooks-available).
 
 2. [Prettier mirror](https://github.com/pre-commit/mirrors-prettier): Checks for code formatting issues within Javascript files and fixes if errors found, like indentation, extra lines and spaces, invalid syntax, etc .
+
+#Installation
+Install pre-commit package manager.
+    pip install pre-commit
```
Comment (dalmia) [inline docs/PRE-COMMIT.md:16]: Use code-blocks for better readability. Refer to `docs/INSTALLATION.md` for this.

File: docs/PRE-COMMIT.md [hunk 3]
```diff
@@ -10,3 +10,13 @@ The pre-commit hook in this repository uses various plugins to run different kin
...
+
+You can install it in your codespace using this: https://github.com/avantifellows/plio-frontend/blob/master/docs/PRE-COMMIT.md.
```
Comment (dalmia) [inline docs/PRE-COMMIT.md:18]: see how to add hyperlink to a text itself on markdown so that one could simply click on `this`.

Comment (dalmia) [inline docs/PRE-COMMIT.md:18]: the user is already in this file right? So, you do not need to link back the same file again!

File: docs/PRE-COMMIT.md [hunk 4]
```diff
@@ -10,3 +10,27 @@ The pre-commit hook in this repository uses various plugins to run different kin
 1. [Pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks): Checks and fixes basic linting issues. All available hooks for pre-commit can be found [here](https://github.com/pre-commit/pre-commit-hooks#hooks-available).
 
 2. [Prettier mirror](https://github.com/pre-commit/mirrors-prettier): Checks for code formatting issues within Javascript files and fixes if errors found, like indentation, extra lines and spaces, invalid syntax, etc .
+
+### Installation
+1. Install pre-commit
+    Use `pip` to install pre-commit
+    ```sh
+    pip install pre-commit
+    ```
```
Comment (deepansh96) [inline docs/PRE-COMMIT.md:19]: If you see how this section gets rendered, you'll notice that "Install pre-commit" and "User `pip` to ...` are not in a separate line.
Screenshot 2022-01-20 at 1 55 16 PM

Comment (deepansh96) [reply docs/PRE-COMMIT.md:19]: You can see how a markdown file will be rendered by click on the view file button. see reference image below. 
Screenshot 2022-01-20 at 1 56 35 PM

**Review by deepansh96 — CHANGES_REQUESTED** [review]

**Review by deepansh96 — APPROVED** [review]

**Pull request merged**